Classes/Domain/Model/Changes/MoveAfter.php

```diff
@@ -71,14 +71,20 @@ public function apply(): void
             $hasEqualParentNode = $parentNode->aggregateId
                 ->equals($parentNodeOfPreviousSibling->aggregateId);
 
-
             $contentRepository = $this->contentRepositoryRegistry->get($subject->contentRepositoryId);
-
+            $rawMoveNodeStrategy = $this->getNodeType($this->subject)?->getConfiguration('options.moveNodeStrategy');
```

Comment:
this btw. should still be part of the schema  @mhsdesign in Neos.Neos as otherwise you cannot validate it....

Reply:
irigh zrk who actually uses the validator 😅 but if you think so ... are the validation things merged? then it can live here in the ui ... other wise id say no its not worth it. its not something anyone is gonna use.

Reply:
😂 I think we cannot merge schemas? Lets skip it for now, but IMHO would be good to have somewhere...